public/index.js

```diff
@@ -1,34 +1,29 @@
 (function() {
 
-  // const message = document.getElementsByClassName('message')[0];
-  // const red = document.getElementsByClassName('red')[0];
-  // const blue = document.getElementsByClassName('blue')[0];
-  // const yellow = document.getElementsByClassName('yellow')[0];
-
+  const redBtn = document.getElementById('red-button');
+  const blueBtn = document.getElementById('blue-button');
+  const yellowBtn = document.getElementById('yellow-button');
   const body = document.querySelector('body');
 
-  body.style.backgroundColor = "cyan";
```

Comment:
This line was originally here so that if the path to the javascript file had been set up correctly the screen's colour would change. As long as the javascript program does something that demonstrates it's now working should be fine. A simple alert() might also be an option.